calamari-common/calamari_common/remote/mon_remote.py

```diff
@@ -1057,3 +1065,30 @@ def tail(subpath, n_lines):
     p = subprocess.Popen(["tail", "-n", str(n_lines), path], stdout=subprocess.PIPE)
     stdout, stderr = p.communicate()
     return stdout
+
+
+def cluster_stats(cluster_name):
+    import rados
+
+    cluster = rados.Rados(name=RADOS_NAME, clustername=cluster_name, conffile='')
+    cluster.connect()
+
+    result = cluster.get_cluster_stats()
+
+    return result
+
+
+def pool_stats(cluster_name, pool_id):
+    import rados
+
+    cluster = rados.Rados(name=RADOS_NAME, clustername=cluster_name, conffile='')
+    cluster.connect()
+
+    try:
+        pool = cluster.pool_reverse_lookup(pool_id)
+    except Exception as e:
+        raise RuntimeError("pool lookup: " + str(e))
+
+    result = cluster.open_ioctx(pool).get_stats()
```

Comment:
@b-ranto be sure to close this http://docs.ceph.com/docs/master/rados/api/python/#input-output-context